api/test/unit/services/auth.service.spec.ts

```diff
@@ -552,34 +795,6 @@ describe('Testing auth service', () => {
     });
   });
 
-  it('should error when trying to request mfa code, but mfa disabled', async () => {
```

Comment:
Is this not a valid use case anymore?

Reply:
Yeah, with ReCaptcha we don't have a sense of MFA being "disabled"